kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -934,21 +951,19 @@ def test_pod_template_file(
         pod_mock.status.phase = "Succeeded"
         await_pod_completion_mock.return_value = pod_mock
         context = create_context(k)
-        with self.assertLogs(k.log, level=logging.DEBUG) as cm:
+        with caplog.at_level(logging.DEBUG, logger="airflow.task"):
             k.execute(context)
-            expected_line = textwrap.dedent(
-                """\
-            DEBUG:airflow.task.operators:Starting pod:
-            api_version: v1
-            kind: Pod
-            metadata:
-              annotations: {}
-              cluster_name: null
-              creation_timestamp: null
-              deletion_grace_period_seconds: null\
-            """
-            ).strip()
-            assert any(line.startswith(expected_line) for line in cm.output)
+        expected_line = (
```

Comment:
Due to the different log format of `caplog`, I decided to reformat this to a regex search instead of the `any(...` logic. This was easier to do with this string format instead of the multiline.